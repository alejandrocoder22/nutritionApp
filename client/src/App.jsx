
import { UserContextProvider } from './context/userContext'
import MainRoutes from './router/MainRoutes'

function App () {
  return (
    <UserContextProvider>
      <MainRoutes />
    </UserContextProvider>
  )
}

export default App
