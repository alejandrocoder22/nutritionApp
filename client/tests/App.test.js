
import { render } from '@testing-library/react'
import App from '../src/App'

describe('Pruebas en <App/>', () => {
    test('Debe de hacer match con el snapshot', () => {
      const { container } =  render(<App/>)
      expect(container).toMatchSnapshot()
    })
})