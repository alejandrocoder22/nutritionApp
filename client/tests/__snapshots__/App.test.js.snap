// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <App/> Debe de hacer match con el snapshot 1`] = `
<div>
  <div
    class="App"
  >
    <h1>
      Hola
    </h1>
  </div>
</div>
`;
